File: programmers/Level1/CPP/2016년.cpp
#include <string>
using namespace std;

int month[12]={31,29,31,30,31,30,31,31,30,31,30,31};

string solution(int a, int b) {
    
    int day = 0;
    for(int i=0; i < a - 1; i++){
        day += month[i];
    }
    day+=b;
    day%= 7;
    
    string answer = "";

    switch(day){
        case 1: answer = "FRI"; break;
        case 2: answer = "SAT"; break;
        case 3: answer = "SUN"; break;
        case 4: answer = "MON"; break;
        case 5: answer = "TUE"; break;
        case 6: answer = "WED"; break;
        case 0: answer = "THU"; break;
    }
    return answer;
}

File: programmers/Level1/JAVA/2016년.java
class Solution {
    public String solution(int a, int b) {
        String answer = "";
        
        int arr[] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        
        int sum = 0;
        for(int i = 0; i< a-1; i++){
            sum += arr[i];
        }
        
        sum += b;
        sum %= 7;
        
        if(sum == 0) answer  = "THU";
        else if(sum == 1) answer  = "FRI";
        else if(sum == 2) answer  = "SAT";
        else if(sum == 3) answer  = "SUN";
        else if(sum == 4) answer  = "MON";
        else if(sum == 5) answer  = "TUE";
        else if(sum == 6) answer  = "WED";
          
        return answer;
    }
}
